server/src/crew/positions.json

```
@@ -2,418 +2,387 @@
-  "Casm Operator",
   "Cellist",
   "Central Intelligence Officer",
   "Chaos Physics Lab Supervisor",
   "Chaos Physics Teacher",
-  "Chaos Theory Specialist",
   "Chaplain",
-  "Chaplain Aide",
   "Chef",
   "Chief Decryption Officer",
+  "Chief Legal Officer",
+  "Chief Medical Officer",
   "Chief Phaser Specialist",
   "Chief Probes Officer",
   "Chief Programmer",
   "Chief Railgun Specialist",
   "Chief Torpedo Specialist",
   "Clerk",
-  "Client / Server Manager",
   "Client Server Manager",
   "Cloaking Control Officer",
-  "Cloaking Device Understudy",
   "Cloaking Manager",
-  "Cloning Research Understudy",
+  "Cloning Researcher",
   "Cloning Teacher",
-  "Co2 Scrubber Control Officer",
-  "Communicaitons Crew Department Head",
+  "CO2 Scrubber Control Officer",
+  "Communications Department Head",
   "Communications Antenna Operator",
   "Communications Antenna Understudy",
   "Communications Array Understudy",
   "Communications Chief",
-  "Communications Crew Department Head",
-  "Communications Diagnostic Understudy",
+  "Communications Diagnostic Officer",
   "Communications Officer",
-  "Communications Specialist",
-  "Communications Sub Department Head",
+  "Communications Officer Understudy",
   "Communications Sub-department Head",
-  "Compressor Operator",
-  "Computer Control",
-  "Computer Core Chief",
+  "Computer Control Department Head",
+  "Computer Core Debugger",
   "Computer Core Maintenance",
-  "Computer Reinitilation Station",
-  "Contractor",
-  "Cook Understudy",
+  "Computer Reinitiation Station",
+  "Coroner",
+  "Cook"
+  "Cook Assistant",
   "Counselor",
-  "Crew Development Officer",
-  "CRM-114 Chief",
+  "Crew Dispatch Officer",
+  "Crime Scene Investigator",
   "Cryonic Specialist",
-  "Cryonic Teacher",
-  "Cryonic Understudy",
+  "Cryonic Specialists Understudy",
+  "Cryonic Researcher",
   "Cultural Studies Attendant",
+  "Custodial Crew Department Head",
   "Dance Instructor",
   "Data Storage",
-  "Decoder",
+  "Deccryptologist",
   "Decontamination Officer",
   "Defense Specialist",
   "Defensive Sail Control",
-  "Defensive Sail Understudy",
   "Deflector Control",
   "Deflector Shield Grid Operator",
-  "Deflector Shield Understudy",
+  "Deflector Shield Grid Understudy",
   "Deputy Chief Engineer",
-  "Deputy Chief Of Operations",
+  "Deputy Chief of Operations",
   "Deputy Communications Officer",
-  "Deputy Maintenance Specialist",
+  "Deputy Maintenance Officer",
+  "Deputy Medical Officer",
   "Deputy Navigations Officer",
   "Deputy Phaser Specialist",
   "Deputy Quartermaster",
   "Deputy Railgun Specialist",
   "Deputy Tactical Officer",
   "Deputy Torpedo Specialist",
-  "Deuterium Control Understudy",
-  "Deuterium Loader",
+  "Deuterium Control Officer",
   "Deuterium Processing",
   "Deuterium Specialist",
-  "Diltihium Storage Understudy",
-  "Director Of Encryption/Decryption Services",
+  "Diltihium Storage Monitor",
+  "Director Of Cyptology Services",
```

Cryptography is more accurate.
```suggestion
  "Director Of Cryptography Services",
```